mantidimaging/core/io/saver.py

```diff
@@ -102,14 +102,17 @@ def save(images: Images,
     make_dirs_if_needed(output_dir, overwrite_all)
 
     # Define current parameters
+    min_value = images.data.min()
+    offset = abs(min_value)
```

Comment:
I don't think should be abs().

If `min_value` is positive, either `offset` should be zero or rescale should shift the values down.